Dockerfile.test.rhel7

```diff
@@ -0,0 +1,85 @@
+# Dockerfile.test.rhel
+#
+# Dockerfile for running tests on RHEL7. To install Python3.9 no RHEL
+# we download Python3.9 configure and build the source
+# The Centos repo files enables access to RPMs required by pip and PyInstaller
+# that are not available on the RHEL7 built-in repos.
+
+# Base image is RHEL7
+FROM registry.access.redhat.com/ubi7/ubi
+
+# Set the install dir env param
+ENV INSTALL_DIR=/opt/conjur-api-python3
+
+# Copy below repo files to tum repo dor to make rpms available
+COPY ./test/CentOS-SCLo-scl-rh.repo \
+     ./test/CentOS-7-Linux-AppStream.repo \
+     /etc/yum.repos.d/
+
+# Copy public keys for repo GPG check
+RUN curl https://raw.githubusercontent.com/sclorg/centos-release-scl/master/centos-release-scl/RPM-GPG-KEY-CentOS-SIG-SCLo \
+  > /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-SCLo \
+  && curl -L https://www.centos.org/keys/RPM-GPG-KEY-CentOS-7 > /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 \
+  # Import public keys for repo GPG check
+  && gpg --import \
+            /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 \
+            /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-SCLo
+
+# Enable yum repos
+RUN yum-config-manager --enable \
+                         ubi-server-rhscl-7-rpms \
+                         ubi-7-server-optional-rpms
+
+RUN yum \
+    --disableplugin=subscription-manager \
+    install -y  bash \
+      wget \
+      binutils \
+      yum-utils \
+      gcc gcc-c++ make \
+      curl \
+      jq \
+      libffi-devel \
+      gnome-keyring \
+      procps \
+      zlib-devel \
+      bzip2 \
+      wget \
+      openssl-devel \
+      bzip2-devel \
+      dbus-x11 \
+      git \
+      rh-python38-python-devel \
+      centos-release-scl \
+      devtoolset-7 \
+      && yum --disableplugin=subscription-manager clean all
+
+# Configure generate unique machne-id file required by dbus-11
+RUN dbus-uuidgen > /var/lib/dbus/machine-id
+
+RUN mkdir -p $INSTALL_DIR
+WORKDIR $INSTALL_DIR
+
+# Copy Python requirements file
+COPY ./requirements.txt $INSTALL_DIR/
+
+# Install Pthon 3.9.7 using pyenv, wheel and required libs
+ENV PYENV_ROOT="/root/.pyenv"
+ENV PATH="$PYENV_ROOT/bin:$PYENV_ROOT/shims:$PATH"
+
+RUN curl -L -s https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash \
+    && eval "$(pyenv init --path)" \
+    && pyenv install 3.9.7 \
+    && pyenv global 3.9.7 \
+    && pip install wheel \
+    && pip install -r requirements.txt
+
+# Copy script files
+COPY ./bin/build_integrations_tests_runner ./test/configure_test_executor.sh /
+COPY . $INSTALL_DIR
+
+# Required to build PyInstaller
```

Comment:
Are we building pyinstaller or using?

Reply:
From what I saw in the log we're building it.

Reply:
which log and where?

Reply:
@eranha